README.md

```diff
@@ -6,12 +6,62 @@ Web service wrapper for [fqm-execution](https://github.com/projecttacoma/fqm-exe
+| option                 |  type   | optional? | description                                                                           |
+| :--------------------- | :-----: | :-------: |           :-------------------------------------------------------------------------- |
+| includeClauseResults   | boolean |    yes    |                                  Option to include clause results. Defaults to false. |
+| includePrettyResults   | boolean |    yes    |             Option to include pretty results on statement results. Defaults to false. |
+| includeHighlighting    | boolean |    yes    |                   Include highlighting in MeasureReport narrative. Defaults to false. |
+| measurementPeriodStart | string  |    yes    | Start of measurement period. Defaults to the period provided in the `Measure` resource|
+| measurementPeriodEnd   | string  |    yes    |  End of measurement period. Defaults to the period provided in the `Measure` resource |
+| calculateSDEs          | boolean |    yes    |                Include Supplemental Data Elementss in calculation. Defaults to false. |
+| calculateHTML          | boolean |    yes    |                           Include HTML structure for highlighting. Defaults to false. |
+
+These types will be passed through to the calculation service, and will be used in the execution of the measure.
+
+### Request Outputs
+
+Each request output has a different output format, based on the data being conveyed:
+* `/calculate`: an array of `ExecutionResult` objects. Each `ExecutionResult` object contains:
+    * `patient`: the ID of the patient this object is associated with
+    * `detailedResults`, a map of `group`s to an object which contains population-, statement-, and clause-level results for that group, as well as the html
+      for that group (if requested by the provided options)
+    * `evaluatedResource`: an array of the FHIR resources used in the calculation of the measure for this patient
+    * `supplementalData`: if requested by the provided options, an array containing the raw and HTML structure results for each supplemental data element in the measure, 
+      calculated for the specified patient
+* `/calculateRaw`: a `cql.Result` object, which contains: 
+   * `patientResults`: A map of patient IDs to Statement-level results for that patient
+   * `localIdPatientResultsMap`: A map of patient IDs to the raw results from every clause in every `Library` in the `Measure` resource.
+   * `patientEvaluatedRecords`: A map of patient IDs to the evaluated records for that patient
+* `/Measure/$care-gaps`: a FHIR `Bundle` resource, which contains:
+   * a `Composition` resource containing the actual Gaps in Care report
+   * a `MeasureReport` resource containing the individual measure results for the patient passed in
+   * a set of `DetectedIssue` resources, one for each gap in the measure. Each `DetectedIssue` resource will contain one or more
```

Comment:
It would be useful to clarify the purpose of DetectedIssue and GuidanceResponse here (e.g. DetectedIssue is a gap, GuidanceResponse is an opportunity to close the gap)

Reply:
Yeah, I'll link to [this wiki page](https://github.com/projecttacoma/fqm-execution/wiki/Gaps-In-Care) in fqm-execution

Reply:
We need to change the "near miss" language on the wiki, but I don't think that will block approval of the PR